include/nbl/builtin/hlsl/blit/blit.hlsl

```diff
@@ -0,0 +1,174 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_BLIT_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BLIT_INCLUDED_
+
+
+#include <nbl/builtin/hlsl/multi_dimensional_array_addressing/multi_dimensional_array_addressing.hlsl>
+#include <nbl/builtin/hlsl/blit/parameters.hlsl>
+#include <nbl/builtin/hlsl/blit/common.hlsl>
+#include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
+#include <nbl/builtin/hlsl/spirv_intrinsics/core.hlsl>
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace blit
+{
+
+// p in input space
+int32_t3 getMinKernelWindowCoord(NBL_CONST_REF_ARG(int32_t3) p, NBL_CONST_REF_ARG(float3) minSupport)
+{
+	return int32_t3(ceil(p - float3(0.5f, 0.5f, 0.5f) + minSupport));
+}
```

Comment:
do we use this function anywhere else, and if no why not fold it into the function below